hazelcast/src/test/java/com/hazelcast/internal/ascii/RestTest.java

```diff
@@ -239,4 +240,31 @@ public void run()
             }
         });
     }
+
+    @Test
+    public void testListNodes() throws IOException {
+        final HazelcastInstance instance1 = Hazelcast.newHazelcastInstance(config);
+        HTTPCommunicator communicator = new HTTPCommunicator(instance1);
+        HazelcastTestSupport.waitInstanceForSafeState(instance1);
+        String result = String.format("{\"status\":\"success\" \"response\":\"[%s]\n%s\n%s\"}",
+                instance1.getCluster().getLocalMember().toString(),
+                BuildInfoProvider.getBuildInfo().getVersion(),
+                System.getProperty("java.version"));
```

Comment:
because hazelcast rest api returns java-version..
While testing, i have to add that version.